### ISSUE TITLE
outlier label to an observation with low outlier score

### ISSUE THREAD

**Issue by rtmatx** [scikit-learn-contrib/hdbscan#185]
Hi,

I ran hdbscan on some dataset and computed the membership vector and outlier score for all the observations. I noticed that some of the observations which are marked as outliers, have lower GOSH outlier score than compared to observations which are marked to a cluster label. Is that expected behavior and if so what can be the conceptual reasons behind label an observation as outlier even-though it has lower GOSH outlier score?

Brief info on the image attached. I have created a dataframe with membership vectors, outlier_score, cluster label. I have 16 clusters for the dataset and hence 16 integer labels for dataframe columns. 

![membership vector](https://user-images.githubusercontent.com/23462596/37301527-cc35c910-25f6-11e8-96fe-d163773cc4e8.png)

**Comment by lmcinnes** [comment]
No, the membership vector is an independent algorithm. I suggest you refer to the original paper by Campello et al. for the definition of GLOSH and how to interpret it. Sorry I can't be more help right now.
